crates/dead-man-switch-web/src/main.rs

```diff
@@ -221,7 +222,9 @@ async fn handle_login(
     user_password.zeroize();
 
     if is_valid {
-        let updated_jar = jar.add(Cookie::new("auth", "true"));
+        let mut cookie = Cookie::new("auth", "true");
+        cookie.set_max_age(Some(CookieDuration::days(30)));
+        let updated_jar = jar.add(cookie);
```

Comment:
Use the example code in https://docs.rs/axum-extra/latest/axum_extra/extract/cookie/struct.PrivateCookieJar.html#method.add without adding the extra `cookie` dep.